inst/pages/21_microbiome_community.qmd

```diff
@@ -223,6 +223,55 @@ pheatmap(mat, annotation_row = taxa_clusters,
          color = colors)
 ```
 
+### Composition heatmap using the NeatMap method {#sec-composition-neatmap}
+
+Another method to visualize community composition is by plotting a NeatMap, which means we use radial theta sorting when plotting the heatmap [@Rajaram2010]. The `getNeatOrder` function in the `miaViz` package allows us to achieve this. This method sorts data points based on their angular position in a 2D space, typically after an ordination technique such as PCA or NMDS has been applied.
+
+The `getNeatOrder` method calculates the angle (theta) for each point relative to the centroid and sorts data points based on these theta values in ascending order. This approach preserves the relationships between data points according to the ordination method's spatial configuration, rather than relying on hierarchical clustering.
+
+Now we can create a NeatMap using our original tse object.
+
+```{r neatmap}
+library(mia)
+library(miaViz)
+library(ComplexHeatmap)
+library(circlize)
+
+# Add a pseudocount to the counts data
+assay(tse, "counts") <- assay(tse, "counts") + 1
+
+# Transform the samples into relative abundances using CLR
+tse <- transformAssay(tse, assay.type = "counts", method = "clr", MARGIN = "samples", name = "clr")
+
+# Transform the features (taxa) into zero mean, unit variance (z transformation)
+tse <- transformAssay(tse, assay.type = "clr", method = "z", MARGIN = "features", name = "z")
```

Comment:
does this work now with the latest mia version from github as I think that "z" was updated to "scale"?

Reply:
yes, although it's "standardize" not scale

Reply:
all updated in the new PR

Reply:
Resolve open comments that have been solved.

Reply:
Ok great!